hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/converter/UpdateConverter.java

```diff
@@ -35,19 +37,23 @@
   private final List<String> partitionPathFields;
   private final List<String> recordKeyFields;
   private final int minPayloadSize;
+  private final String preCombineField;
+  private final int preCombineFieldValue;
 
-  public UpdateConverter(String schemaStr, int minPayloadSize, List<String> partitionPathFields,
-      List<String> recordKeyFields) {
+  public  UpdateConverter(String schemaStr, int minPayloadSize, List<String> partitionPathFields,
+      List<String> recordKeyFields, String preCombineField, int preCombineFieldValue) {
```

Comment:
Can you explain why we need to introduce these 2 fields ? Also, please comment why the preCombineFieldValue is "int" ?